--- esound-0.2.37/esd.conf.spawntime	2007-01-02 00:56:06.000000000 +0100
+++ esound-0.2.37/esd.conf	2007-03-30 14:36:36.000000000 +0200
@@ -4,7 +4,7 @@
 # you should start it from the session controller, not some random
 # app inside.
 auto_spawn=0
-spawn_options=-terminate -nobeeps -as 2
-spawn_wait_ms=100
+spawn_options=-terminate -nobeeps
+spawn_wait_ms=300
 # default options are used in spawned and non-spawned mode
-default_options=
+default_options=-as 1
